app/services/chat_service.rb

```diff
@@ -38,8 +38,8 @@ def self.configure_url(chat)
                                                                     host: Rails.configuration.web_app_hostname)
   end
 
-  def send_update(chat, msg)
-    @telegram_api.sendMessage(chat_id: chat.telegram_chat_id, text: msg)
+  def send_update(chat, msg, disable_web_preview = false)
+    @telegram_api.sendMessage(chat_id: chat.telegram_chat_id, text: msg, disable_web_page_preview: disable_web_preview)
```

Comment:
I like it a lot! I was going to research myself if it's possible to disable preview.

I think, it makes sense to disable preview for all other GitHub notifications. Could you please do that since you're already here?